src/app/login/login.component.ts

```diff
@@ -25,6 +25,14 @@ export class LoginComponent {
   model = { name: '', password: '', repeatPassword: '' };
   message = '';
 
+  ngOnInit() {

```

Comment:
you must also check if admin is exist or not

Reply:
@Rupesh87 I think "no user" exists means no admin

Reply:
@yaoxing1990 No documents in the `_users` database does not mean no admin.  CouchDB stores admins in the main configuration database.